### PR TITLE
about page design

### PR DESCRIPTION
**Pull request by Lise23**
#### Before submitting, make sure:
- [ ] You have rebased your branch
- [ ] You describe your changes
- [ ] You describe How to test
- [ ] I work on the same file as one of my colleague which one?

### CODE REVIEW

**Comment by Lise23** [comment]
design request by amaury done :
images taller
more margin inside cards
margin on the top and bottom of the page

File: src/about/About.tsx [hunk 1]
```diff
@@ -1,61 +1,64 @@
...
+            photo: PhotoLise,
+            identity: 'Lise Guichard',
+            description:
+                'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aenean vel convallis mauris, ac bibendum nibh. Nulla porta, sapien quis interdum sodales, nunc lorem aliquet nulla, non consequat tortor ligula et urna. Fusce nunc dolor, vehicula sit amet ex at, pellentesque ultricies ante. Suspendisse vel leo vel nisl finibus feugiat eget at risus. Suspendisse potenti. Phasellus nec luctus leo. Suspendisse quis dui aliquam nunc malesuada consectetur.',
+            onRight: true,
+        },
+    ];
+    return (
+        <div className="h-fit">
+            <div className="h-8"/>
```
Comment (mathildegrimal) [inline src/about/About.tsx:50]: ça le fait mais pourquoi pas un mt-8 sur le container plutôt ?

File: src/about/About.tsx [hunk 2]
```diff
@@ -1,61 +1,64 @@
...
+            {profiles.map(({photo, identity, description, onRight}) => (
+                <Profile
+                    photo={photo}
+                    identity={identity}
+                    description={description}
+                    onRight={onRight}
+                />
+            ))}
+            <div className="h-16"/>
```
Comment (mathildegrimal) [inline src/about/About.tsx:59]: du coup t'as essayé sur le container ou il y a le h-fit de mettre mt-8 et mb-16 ?

**Review by mathildegrimal — COMMENTED** [review]
goooooooood

**Review by mathildegrimal — APPROVED** [review]

**Pull request merged**